### PR TITLE
Sync tsvs from resstock-estimation

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Companion: https://github.com/NREL/resstock-estimation/pull/125

Remaining changes:
* `project_national`
  * [x] Dishwasher: accept these changes as only column order has changed
  * [x] Geometry Building Type ACS: accept these changes as only column order has changed
  * [x] HVAC Has Shared System: accept these changes as only column order has changed
  * [x] Heating Fuel: accept these changes as only column order has changed
  * [x] Misc Extra Refrigerator: accept these changes and update options lookup for EF 19.9
  * [x] Misc Pool Heater: accept changes as only column order has changed
  * [x] Refrigerator: accept these changes
  * [x] Vacancy Status: accept these changes as only column order has changed
  * [x] Vintage: accept these changes as only column order has changed

* `project_testing`
  * [x] Dishwasher: accept these changes as only column order has changed
  * [x] Geometry Building Type ACS: accept these changes as only column order has changed
  * [x] HVAC Cooling Efficiency: probably don't need the extra dependencies
  * [x] HVAC Has Shared System: probably don't need the extra dependencies
  * [x] HVAC Shared Efficiencies: probably don't need the extra dependencies
  * [x] Misc Extra Refrigerator: accept these changes
  * [x] Misc Freezer: accept changes as only column order has changed
  * [x] Refrigerator: accept these changes

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [x] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [x] No unexpected regression test changes
- [x] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: project_national/housing_characteristics/Ceiling Fan.tsv [hunk 1]
```diff
@@ -1,5 +1,3 @@
-Dependency=Vacancy Status	Option=None	Option=Standard Efficiency	Option=Standard Efficiency, No usage
```
Comment (afontani) [inline project_national/housing_characteristics/Ceiling Fan.tsv:1]: We should keep the added dependency on Vacancy Status

File: project_national/housing_characteristics/Corridor.tsv [hunk 2]
```diff
@@ -1,7 +1,3 @@
-Dependency=Geometry Building Type RECS	Option=Not Applicable	Option=Double-Loaded Interior	Option=None	Option=Single Exterior Front	Option=Double Exterior
```
Comment (afontani) [inline project_national/housing_characteristics/Corridor.tsv:1]: We should keep the dependency with Geometry Building Type RECS. It doesn't cause an error but stops the results.csv saying that a Mobile Home, Single-Family Detached, and Single-Family Attached have double-loaded corridors.

File: project_national/housing_characteristics/Insulation Unfinished Attic_ew.txt [hunk 3]
```diff
@@ -0,0 +1,455 @@
+Dependency=Geometry Building Type RECS	Dependency=Location Region	Dependency=Vintage	Option=None	"Option=Uninsulated, Vented"	"Option=Ceiling R-7, Vented"	"Option=Ceiling R-13, Vented"	"Option=Ceiling R-19, Vented"	"Option=Ceiling R-30, Vented"	"Option=Ceiling R-38, Vented"	"Option=Ceiling R-49, Vented"	resstock_probability
```
Comment (afontani) [inline project_national/housing_characteristics/Insulation Unfinished Attic_ew.txt:1]: This file should be deleted from `resstock-estimation`

File: project_national/housing_characteristics/Misc Extra Refrigerator.tsv [hunk 4]
```diff
@@ -1,9 +1,11 @@
...
+Mobile Home	0.005345	0.000903	0.003979	0.026358	0.045026	0.004034	0.914355	533.000000	6842068.000000
+Multi-Family with 2 - 4 Units	0.002677	0.000391	0.003944	0.018584	0.032607	0.003038	0.938759	921.000000	8975958.000000
+Multi-Family with 5+ Units	0.000268	0.000045	0.000839	0.003599	0.020445	0.002291	0.972513	1900.000000	18886024.000000
+Single-Family Attached	0.011469	0.001771	0.013461	0.041609	0.078049	0.004957	0.848683	887.000000	6698326.500000
+Single-Family Detached	0.033185	0.005329	0.028505	0.082042	0.157780	0.017258	0.675901	7743.000000	71332376.000000
 # Created by: sources\recs\2009\tsv_maker.py
 # Source: Constructed using U.S. EIA 2009 Residential Energy Consumption Survey (RECS) microdata.
-# Source: Age of refrigerator converted to efficiency levels using ENERGYSTAR shipment-weighted efficienciesby year.
\ No newline at end of file
+# Source: Age of refrigerator converted to efficiency levels using ENERGYSTAR shipment-weighted efficienciesby year data from Home Energy Score: http://hes-documentation.lbl.gov/. Check the comments in: HES-Refrigerator_Age_vs_Efficiency.tsv
```
Comment (afontani) [inline project_national/housing_characteristics/Misc Extra Refrigerator.tsv:9]: White space `efficienciesby` --> `efficiencies by`. Needs to be fixed in resstock-estimation

File: project_national/housing_characteristics/Misc Extra Refrigerator.tsv [hunk 5]
```diff
@@ -1,9 +1,11 @@
-Dependency=Geometry Building Type RECS	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 6.7	Option=None	sample_count	sample_weight
-Mobile Home	0.002561	0.007348	0.031474	0.041683	0.002561	0.914373	533.000000	6842068.000000
-Multi-Family with 2 - 4 Units	0.001227	0.007565	0.019774	0.031681	0.001227	0.938526	921.000000	8975958.000000
-Multi-Family with 5+ Units	0.000121	0.001675	0.004964	0.020557	0.000121	0.972562	1900.000000	18886024.000000
-Single-Family Attached	0.005188	0.024873	0.046278	0.072301	0.005188	0.846172	887.000000	6698326.500000
-Single-Family Detached	0.014856	0.054054	0.086323	0.155556	0.014856	0.674355	7743.000000	71332376.000000
+Dependency=Geometry Building Type RECS	Option=EF 6.7	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 19.9	Option=None	sample_count	sample_weight
```
Comment (afontani) [inline project_national/housing_characteristics/Misc Extra Refrigerator.tsv:1]: Seems to be an updated file that wasn't pulled over to ResStock. The distributions are different, not just out of order. There is also an added EF 19.9 option that is not in `develop`.

File: project_national/housing_characteristics/Refrigerator.tsv [hunk 6]
```diff
@@ -1,24 +1,11 @@
...
+Mobile Home	0.054273	0.008690	0.035322	0.195062	0.617901	0.088753	0.000000	533.000000	6842068.000000
+Multi-Family with 2 - 4 Units	0.030439	0.005122	0.041124	0.212894	0.642879	0.067543	0.000000	921.000000	8975958.000000
+Multi-Family with 5+ Units	0.032757	0.005397	0.033313	0.179132	0.681388	0.065999	0.002014	1900.000000	18886024.000000
+Single-Family Attached	0.035360	0.005510	0.035532	0.187268	0.623666	0.111545	0.001119	887.000000	6698326.500000
+Single-Family Detached	0.028144	0.004337	0.037080	0.180561	0.606475	0.141354	0.002049	7743.000000	71332376.000000
 # Created by: sources\recs\2009\tsv_maker.py
 # Source: Constructed using U.S. EIA 2009 Residential Energy Consumption Survey (RECS) microdata.
-# Source: Age of refrigerator converted to efficiency levels using ENERGYSTAR shipment-weighted efficiencies by year.
+# Source: Age of refrigerator converted to efficiency levels using ENERGYSTAR shipment-weighted efficienciesby year data from Home Energy Score: http://hes-documentation.lbl.gov/. Check the comments in: HES-Refrigerator_Age_vs_Efficiency.tsv
```
Comment (afontani) [inline project_national/housing_characteristics/Refrigerator.tsv:9]: White space `efficienciesby` --> `efficiencies by`. Needs to be fixed in resstock-estimation.

File: project_national/housing_characteristics/Refrigerator.tsv [hunk 7]
```diff
@@ -1,24 +1,11 @@
...
-Multi-Family with 2 - 4 Units	High	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	Average	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	Medium	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	Low	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
-Mobile Home	High	0.024380	0.066827	0.321826	0.562587	0.024380	133.250000	1710517.000000
-Multi-Family with 2 - 4 Units	Average	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	Medium	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	Low	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	High	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
+Dependency=Geometry Building Type RECS	Option=EF 6.7	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 19.9	Option=None	sample_count	sample_weight
```
Comment (afontani) [inline project_national/housing_characteristics/Refrigerator.tsv:1]: These two files are different. I am guessing that the removal of Usage Level and the added EF 19.9 option is the change we want to keep as the efficiencies I think have been updated.

File: project_national/housing_characteristics/source_report.csv [hunk 8]
```diff
@@ -0,0 +1,108 @@
+Parameter,Created by,Source,Assumption
```
Comment (afontani) [inline project_national/housing_characteristics/source_report.csv:1]: We should not copy this file over. Or at least into the `housing_ characteristics` folder.

File: project_testing/housing_characteristics/HVAC Cooling Efficiency.tsv [hunk 9]
```diff
@@ -1,18 +1,3 @@
-Dependency=HVAC Cooling Type	Dependency=HVAC Has Shared System	Option=AC, SEER 8	Option=AC, SEER 10	Option=AC, SEER 13	Option=AC, SEER 15	Option=Heat Pump	Option=None	Option=Room AC, EER 8.5	Option=Room AC, EER 9.8	Option=Room AC, EER 10.7	Option=Room AC, EER 12.0	Option=Shared Cooling
```
Comment (afontani) [inline project_testing/housing_characteristics/HVAC Cooling Efficiency.tsv:1]: I don't know if these dependencies are needed in `project_testing`. May need to test.

File: project_testing/housing_characteristics/HVAC Has Shared System.tsv [hunk 10]
```diff
@@ -1,102 +1,3 @@
-Dependency=Geometry Building Type RECS	Dependency=HVAC Cooling Type	Dependency=HVAC Heating Type	Option=Cooling Only	Option=Heating and Cooling	Option=Heating Only	Option=None
```
Comment (afontani) [inline project_testing/housing_characteristics/HVAC Has Shared System.tsv:1]: I don't know if these dependencies are needed in project_testing. May need to test.

File: project_testing/housing_characteristics/HVAC Shared Efficiencies.tsv [hunk 11]
```diff
@@ -1,26 +1,3 @@
-Dependency=Heating Fuel	Dependency=HVAC Has Shared System	Option=Boiler Baseboards Heating Only, Electricity	Option=Boiler Baseboards Heating Only, Fuel Oil	Option=Boiler Baseboards Heating Only, Natural Gas	Option=Boiler Baseboards Heating Only, Propane	Option=Fan Coil Cooling Only	Option=Fan Coil Heating and Cooling, Electricity	Option=Fan Coil Heating and Cooling, Fuel Oil	Option=Fan Coil Heating and Cooling, Natural Gas	Option=Fan Coil Heating and Cooling, Propane	Option=None
```
Comment (afontani) [inline project_testing/housing_characteristics/HVAC Shared Efficiencies.tsv:1]: I don't know if these dependencies are needed in project_testing. May need to test.

**Review by afontani — COMMENTED** [review]
There seem to be some significant changes. Most of the changes are re-ordering columns. I think when @rajeee implemented the human sort function for options in resstock-estimation. I added a bunch of comments on things that can be changed, need to be fixed and tested. I was not able to view the differences in Heating Fuel, Vintage, and Vacancy Status. But like Geometry Building Type ACS I believe the columns have been reordered from the human sort column.

File: project_national/housing_characteristics/Ceiling Fan.tsv (hunk 1 above)
Comment (joseph-robertson) [reply project_national/housing_characteristics/Ceiling Fan.tsv:1]: @afontani So by keep, do you mean copy resstock's version over to resstock-estimation? Or would some tsv maker code need to be updated in resstock-estimation?

File: project_national/housing_characteristics/Corridor.tsv (hunk 2 above)
Comment (joseph-robertson) [reply project_national/housing_characteristics/Corridor.tsv:1]: @afontani Same question as Ceiling Fan.

File: project_national/housing_characteristics/Misc Extra Refrigerator.tsv (hunk 5 above)
Comment (joseph-robertson) [reply project_national/housing_characteristics/Misc Extra Refrigerator.tsv:1]: @afontani Do we assume that resstock-estimation has the correct version?

Comment (afontani) [reply project_national/housing_characteristics/Misc Extra Refrigerator.tsv:1]: @joseph-robertson: yes.

File: project_national/housing_characteristics/Ceiling Fan.tsv (hunk 1 above)
Comment (afontani) [reply project_national/housing_characteristics/Ceiling Fan.tsv:1]: We just need to copy the "Ceiling Fan.tsv" to resstock-estimation.

Comment (afontani) [reply project_national/housing_characteristics/Ceiling Fan.tsv:1]: We should copy the "Ceiling Fan.tsv" from resstock to resstock-estimation.

File: project_national/housing_characteristics/Corridor.tsv (hunk 2 above)
Comment (afontani) [reply project_national/housing_characteristics/Corridor.tsv:1]: We should copy the "Corridor.tsv" from resstock to resstock-estimation.

File: project_national/housing_characteristics/Refrigerator.tsv (hunk 7 above)
Comment (afontani) [reply project_national/housing_characteristics/Refrigerator.tsv:1]: In this case, resstock-estimation has the correct file.

Comment (joseph-robertson) [reply project_national/housing_characteristics/Refrigerator.tsv:1]: @afontani I think the "None" option needs to be removed, as that option does not exist in the options lookup. Can you show me what needs to be done in the RECS 2009 tsv maker?

File: project_testing/housing_characteristics/HVAC Cooling Efficiency.tsv (hunk 9 above)
Comment (joseph-robertson) [reply project_testing/housing_characteristics/HVAC Cooling Efficiency.tsv:1]: Integrity checks pass without these dependencies.

File: project_testing/housing_characteristics/HVAC Shared Efficiencies.tsv (hunk 11 above)
Comment (joseph-robertson) [reply project_testing/housing_characteristics/HVAC Shared Efficiencies.tsv:1]: Integrity checks pass without these dependencies.

File: project_testing/housing_characteristics/HVAC Has Shared System.tsv (hunk 10 above)
Comment (joseph-robertson) [reply project_testing/housing_characteristics/HVAC Has Shared System.tsv:1]: Integrity checks pass without these dependencies.

**Review by afontani — COMMENTED** [review]
Observed differences in the large files (Heating Fuel, Vacancy Status, and Vintage) are due to the dependency columns being reordered. The changes result in no change to the distributions.

**Review by afontani — APPROVED** [review]

**Pull request merged**